projects/common/zed/zed_system_constr.xdc

```diff
@@ -41,8 +41,8 @@ set_property  -dict {PACKAGE_PIN  AA7   IOSTANDARD LVCMOS33} [get_ports i2s_sdat
 
 # iic
 
-set_property  -dict {PACKAGE_PIN  R7    IOSTANDARD LVCMOS33} [get_ports iic_scl]
-set_property  -dict {PACKAGE_PIN  U7    IOSTANDARD LVCMOS33} [get_ports iic_sda]
+set_property  -dict {PACKAGE_PIN  R7    IOSTANDARD LVCMOS33} [get_ports iic_scl]       ; ## C30  FMC-SCL
+set_property  -dict {PACKAGE_PIN  U7    IOSTANDARD LVCMOS33} [get_ports iic_sda]       ; ## C31  FMC-SDA
```

Comment:
From what I've seen we don't have I2C pins description on other carriers. We could drop these modifications for now and check if we need to update descriptions for all carriers.